README.md

```diff
@@ -1 +1,57 @@
-# hello-rails-back-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello Rails Back End
+
+> This rails api app is just the back end of another app
+
+## Front end repository
+> https://github.com/ulisesedu7/hello-rails-front-end/pull/1
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+- Rspec for testing
+- Git & Github
+
+## Setup
+
+1. You would need to have a computer with ruby, rails and postgresql installed to run it.
+2. Follow these steps on your console to properly clone this repository on your desktop:
+
+```
+$ git clone 'repo_path'
```

Comment:
Good job for your hard work 👏 you have done a great job. In addition to what you have done, I would suggest that you use the UPDATED [README TEMPLATE](https://github.com/microverseinc/readme-template) as provided by Microverse which can be [found here](https://github.com/microverseinc/readme-template). Note that it contains new section and features which helps it to be outstanding 😎 I would highly recommend using it, and following all sections